extensions/scanning/README.md

```diff
@@ -12,7 +12,7 @@ change**
 
 ## Item Properties or Asset Fields
 
-| Field Name       | Type    | Description                                                           |
-| ---------------- | ------- | --------------------------------------------------------------------- |
-| scan:is_original | boolean | Whether the scan is of an original negative (true) or a copy (false). |
-| scan:datetime    | string  | Year and quarter during which the photo was scanned/delivered to us.  |
+| Field Name       | Type    | Description                                                                                                                                                                                                                                                     |
+| ---------------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| scan:is_original | boolean | Whether the scan is of an original negative (true) or a copy (false).                                                                                                                                                                                           |
+| scan:scanned     | string  | Government year quarter start date which the photo was scanned/delivered to us, in UTC. Formatted according to [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6) and [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html). |
```

Comment:
`scanned` description change:

> Date and time the corresponding data was scanned. For LINZ data, this will be the start datetime for the calendar year quarter in which the data was scanned. MUST be formatted according to [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6).

I would like to move the "For LINZ data..." sentence to the composite extension specifically for Historical Imagery once that exists, but let's leave it here for now.

No need to reference ISO 8601 since RFC 3339 already does that and we aren't introducing any additional elements from ISO 8601.